sql/builder/builder_test.go

```diff
@@ -0,0 +1,93 @@
+package builder
+
+import (
+	"testing"
+
+	"github.com/spacemeshos/go-spacemesh/common/types"
+)
+
+func TestFilterFrom_WithSingleFilter(t *testing.T) {
+	operations := Operations{
```

Comment:
NIT: use `t.Parallel()` to speed up tests.
```suggestion
func TestFilterFrom_WithSingleFilter(t *testing.T) {
	t.Parallel()
	operations := Operations{
```